### PR TITLE
refactor(settings): reorganize settings styles

### LINKED ISSUE #271: Tidy up settings menu

### PR DESCRIPTION
**Pull request by johngruen**
@meandavejustice settings look great! just a quick polish pass. 

Fixes #271

### CODE REVIEW

**Comment by johngruen** [comment]
ugh, rebase incoming

File: idea_town/frontend/static-src/styles/modules/_settings.scss [hunk 1]
```diff
@@ -1,62 +1,92 @@
...
+  }
 
-    span {
-        background: white;
-        width: 8px;
-        height: 8px;
-        border-radius: 50%;
-        display: inline-block;
+  span {
+    background: white;
```
Comment (pdehaan) [inline idea_town/frontend/static-src/styles/modules/_settings.scss]: nit: `$white`

File: idea_town/frontend/static-src/styles/modules/_settings.scss [hunk 2]
```diff
@@ -1,62 +1,92 @@
...
-    position: relative;
-    margin-top: 20px;
+  margin-top: 20px;
+  position: relative;
 }
 
 .settings-menu {
+  background: white;
+  border-radius: 3px;
+  border-top: 1px solid white;
```
Comment (pdehaan) [inline idea_town/frontend/static-src/styles/modules/_settings.scss]: nit: also maybe `$white` here and on line 52.

**Comment by pdehaan** [comment]
@johngruen Fixes #271?

**Pull request merged**